packages/navie/src/agents/test-agent.ts

```diff
@@ -79,6 +79,7 @@ export default class TestAgent implements Agent {
       await contentFetcher.applyFileContext(options, options.chatHistory);
     }
 
+    await this.contextService.locationContextFromOptions(options);
     await this.contextService.searchContext(options, tokensAvailable, ['test', 'spec']);
```

Comment:
Is there a place where we just want one or the other? If not, perhaps it could be one method call.